.idea/vcs.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-    <component name="IssueNavigationConfiguration">
-        <option name="links">
-            <list>
-                <IssueNavigationLink>
-                    <option name="issueRegexp" value="#(\d+)" />
-                    <option name="linkRegexp" value="https://github.com/opensearch-project/OpenSearch/pulls/$1" />
-                </IssueNavigationLink>
-                <IssueNavigationLink>
-                    <option name="issueRegexp" value="#(\d+)" />
-                    <option name="linkRegexp" value="https://github.com/opensearch-project/OpenSearch/issues/$1" />
-                </IssueNavigationLink>
-            </list>
-        </option>
-    </component>
+  <component name="IssueNavigationConfiguration">
```

Comment:
Why are we pushing this file @scrawfor99?

Reply:
Sorry looks like a bad style change. I reverted it.

Reply:
Can you remove the change of this file completely? Let's not push it at all

Reply:
Yeah that is what I did.